pulsar-client-cpp/tests/BatchReceivePolicyTest.cc

```diff
@@ -0,0 +1,45 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+#include <gtest/gtest.h>
+#include <pulsar/BatchReceivePolicy.h>
+
+using namespace pulsar;
+
+TEST(BatchReceivePolicyTest, testBatchReceivePolicy) {
+    try {
+        BatchReceivePolicy batchReceivePolicy(-1, -1, -1);
+        FAIL() << "Should be failed.";
+    } catch (const std::invalid_argument& e) {
+        ASSERT_TRUE(true);
+    }
```

Comment:
`ASSERT_TRUE(true)` is meaningless. If you don't care about what `e.what()` returns`, use `ASSERT_THROWS` here. Otherwise, use `ASSERT_TRUE(/* condition about e.what()... */)`.